Create-AzSStorageDashboard.ps1

```diff
@@ -204,6 +224,23 @@ function Get-volumesByType {
 
     $volumesByType = @{}
 
+    
```

Comment:
Seems no need to write the snippet twice. Could move it after `if ($capacityOnly -eq $True) {`.